doc_source/mesh-k8s-integration.md

```diff
@@ -1,10 +1,10 @@
-+  **App Mesh sidecar injector for Kubernetes** – The injector installs as a webhook and injects the following containers into Kubernetes pods that are running in specific, labeled namespaces\. To learn more or contribute, see the [GitHub project](https://github.com/aws/aws-app-mesh-inject)\.
-  + **App Mesh Envoy proxy** –Envoy uses the configuration defined in the App Mesh control plane to determine where to send your application traffic\.\. 
-  + **App Mesh proxy route manager **– The route manager sets up a pod’s network namespace with `iptables` rules that route ingress and egress traffic through Envoy\.
+When you integrate AWS App Mesh with Kubernetes, you manage App Mesh resources, such as meshs, virtual services, virtual nodes and virtual routers, through Kubernetes\. You also automatically add the App Mesh sidecar container images to Kubernetes pod specifications\. This tutorial guides you through the installation of the following open source components that enable this integration:
++ **App Mesh manager for Kubernetes** – The manager is accompanied by the deployment of four Kubernetes custom resource definitions: `mesh`, `virtual service`, `virtual node`, and `virtual router` \. The controller watches for creation, modification, and deletion of the custom resources and makes changes to the corresponding App Mesh `[mesh](https://docs.aws.amazon.com/app-mesh/latest/userguide/meshes.html)`, `[virtual service](https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_services.html)`, `[virtual router](https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_routers.html)` (including `[route](https://docs.aws.amazon.com/app-mesh/latest/userguide/routes.html)`\), and `[virtual node](https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_nodes.html)` resources through the App Mesh API\. To learn more or contribute to the controller, see the [GitHub project](https://github.com/aws/aws-app-mesh-controller-for-k8s)\.
++  **App Mesh sidecar injector for Kubernetes** – The injector installs as a webhook and injects the following containers into Kubernetes pods that are running in specific, labeled namespaces\. The sidecar injector runs inside the App Mesh manager.
+    + **App Mesh Envoy proxy** –Envoy uses the configuration defined in the App Mesh control plane to determine where to send your application traffic\.\.
+    + **App Mesh proxy route manager **– The route manager sets up a pod’s network namespace with `iptables` rules that route ingress and egress traffic through Envoy\. This runs as a Kubernetes init container inside a pod 
```

Comment:
Might not be accurate to say proxy init container "sets up a pod’s network namespace". I think we can just say "Updates 'iptables' rules in pod's network namespace to route ingress and egress pod traffic through Envoy"

Reply:
Hmm, why would it not be accurate to call it an init container. Is it an init container, no?

Reply:
No, the comment was about - "sets up a pod’s network namespace". Init container doesn't do that right, it just updates iptable rules

Reply:
Ah, I see the confusion. Fair point. I'll adjust the wording